multinode-demo/client-throttled.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I'm not super keen on growing `multinode-demo/`.   I'd rather whoever will run `multinode-demo/client-throttled.sh` instead just runs `multinode-demo/client.sh` and supplies the different arguments

Reply:
💯